we take care of building this ourselves in the ebuild so
build settings are properly respected

--- dhcp-4.2.2/bind/Makefile
+++ dhcp-4.2.2/bind/Makefile
@@ -29,6 +29,7 @@
 bindsrcdir=bind-${version}
 
 all:
+disable:
 # Extract the source from the tarball, if it hasn't been already.
 	@if test -d ${bindsrcdir} ; then                    \
 		echo ${bindsrcdir} already unpacked... ;    \
